mu4/scenes/notation/view/playbackcursor.h

```diff
@@ -0,0 +1,52 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2020 MuseScore BVBA and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+#ifndef MU_NOTATIONSCENE_PLAYBACKCURSOR_H
+#define MU_NOTATIONSCENE_PLAYBACKCURSOR_H
+
+#include <QRect>
+#include <QColor>
+
+class QPainter;
+
+namespace mu {
+namespace scene {
+namespace notation {
+class PlaybackCursor
+{
+public:
+    PlaybackCursor();
```

Comment:
It will be generated automatically

Reply:
In fact, I consider it a waste of time to remove the constructors generated by the IDE when the class is created.

Reply:
changed on `PlaybackCursor() = default;`